operators/src/error.rs

```diff
@@ -500,6 +500,10 @@ pub enum Error {
     Bb8Postgres {
         source: bb8::RunError<tokio_postgres::Error>,
     },
+
+    DatasetDeleted {
```

Comment:
add a snafu displaymessage that the dataset could not be accessed because the data was deleted.

Reply:
265eba09b4725423b2064fad2ad936bbf66f69c0